resources/assets/components/pages/AccountPage/Subscriptions/CancelEmailSubscription.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+
+import Spinner from '../../../artifacts/Spinner/Spinner';
+
+const EMAIL_SUBSCRIPTION_STATUS = gql`
+  query EmailSubscriptionStatus($userId: String!) {
+    user(id: $userId) {
+      id
+      emailSubscriptionStatus
+    }
+  }
+`;
+const EMAIL_SUBSCRIPTION_STATUS_MUTATION = gql`
+  mutation EmailSubscriptionStatus(
+    $userId: String!
+    $emailSubscriptionStatus: Boolean!
+  ) {
+    updateEmailSubscriptionStatus(
+      id: $userId
+      emailSubscriptionStatus: $emailSubscriptionStatus
+    ) {
+      id
+      emailSubscriptionStatus
+    }
+  }
+`;
+const CancelEmailSubscription = props => {
+  const userId = props.user.id;
+  const options = { variables: { userId } };
```

Comment:
@Vmack per [Aaron's comment](https://dosomething.slack.com/archives/CUQMU4Q6B/p1593189750008500?thread_ts=1593187616.006000&cid=CUQMU4Q6B), if we add a `refetchQueries: ['EmailSubscriptionsQuery']` here, this should force those individual email topic subscription queries to refresh which should update without a page refresh!
```suggestion
  const options = { variables: { userId }, refetchQueries: ['EmailSubscriptionsQuery'] };
```

Reply:
We'd also need the `EmailSubscriptionItem` to list this query here as a `refetch` so that toggling an individual topic subscription, eventually causes the cancel link to re-appear when they're _resubscribed_ to email. (But also blocked by [this bug](https://dosomething.slack.com/archives/CUQMU4Q6B/p1593190746009400?thread_ts=1593187616.006000&cid=CUQMU4Q6B))
https://github.com/DoSomething/phoenix-next/blob/1194d7d2e0a3187eed2cea8c244ed9fffe41f4a2/resources/assets/components/pages/AccountPage/Subscriptions/EmailSubscriptionItem.js#L41

`const options = { variables: { userId: window.AUTH.id }, refetchQueries: ['EmailSubscriptionStatus` }; `

Reply:
I think we're taking advantage of this now, but just noting for future reference that including `id` on each of these queries should let Apollo take care of updating these two UIs for us, without explicitly tying them to each other!

(This works because Apollo keeps an in-memory representation of each object that's returned from a query with an ID, e.g. `User:5543dfd6469c64ec7d8b46b3`. It then updates that in-memory cache whenever it gets new information with the same ID, which will trigger a re-render of any components that depend on that data).

Reply:
Ooooh I remember you mentioning something like this in that dev roundtable / sprint demo a while back re the causes/subscription components, but I couldn't for the life of me recall explicitly if that would work here 🤦

Reply:
Wait but @DFurnes aren't we indeed including the user ID in the query/mutation above? (Looks like we are in the Topics as well). Should that automatically cause these two to update / is that precisely your point in https://github.com/DoSomething/phoenix-next/pull/2201#discussion_r449085475?

Reply:
Yeah, my understanding is that this should "just work" since we're including the `id`!

Reply:
Hrmmmm we may need to page Houston here 👩‍🚀  👨‍🚀

Reply:
yea I thought so too which is why I'm wondering why it doesn't work

Reply:
@DFurnes @Vmack Quick update per Diego D's bug [noted in Slack](https://dosomething.slack.com/archives/CP2D7UGAU/p1594821771010600?thread_ts=1594657236.003600&cid=CP2D7UGAU). It seems that we're still not getting an automatic refresh of the subscription status even with our explicit `refetchQueries` solution.

But per Dave's point here, I was wondering why this actually might be the case. 

To me, it _seems_ from the Apollo Mutation docs [here](https://www.apollographql.com/docs/react/data/mutations/#updating-a-single-existing-entity) that we'd actually need to return the modified field explicitly within our query in order to update the cache (emphasis mine):

> To do so, the mutation must return the id of the modified entity, _along with the values of the fields that were modified_. Conveniently, mutations do this by default in Apollo Client.

So our issue might just be that we're not including the 'secondarily' modified `emailSubscriptionStatus`, or contrarily the `emailSubscriptionTopics` from each respective query!

Indeed, on my local dev, when I included those fields explicitly, things updated automatically! _Without_ the `refreshQueries` option.

Reply:
oooohh so then that would mean that I need to just get rid of the redundant `refreshQueries` and just add the modified `emailSubscriptionTopic` to each query?

Reply:
Exactly! I think we'd just need to add the `emailSubscriptionTopic` to the `CancelEmailSubscription` mitation since that's the field that's updated as a 'side effect'
https://github.com/DoSomething/phoenix-next/blob/941dea7923cc451678a29926ef051d361e3baa13/resources/assets/components/pages/AccountPage/Subscriptions/CancelEmailSubscription.js#L17-L28

And add the `emailSubscriptionStatus` field to the `EmailSubscriptionTopic` mutation:
https://github.com/DoSomething/phoenix-next/blob/941dea7923cc451678a29926ef051d361e3baa13/resources/assets/components/pages/AccountPage/Subscriptions/EmailSubscriptionItem.js#L19-L32

Reply:
fantastic